utils/lib/mix/tasks/add_git_section.ex

```diff
@@ -0,0 +1,78 @@
+defmodule Mix.Tasks.AddGitSection do
+  @moduledoc "The add git section mix task: `mix help add_git_section`"
+  @shortdoc "Adds git commit section to files"
+  
+  use Mix.Task
+
+  @ignore_reading_files [
+    "code_editors.livemd",
+    "command_line.livemd",
+    "git.livemd",
+    "livebook.livemd"
+  ]
+  @ignore_exercise_files [
+    "command_line_family_tree.livemd",
+    "github_collab.livemd",
+    "github_engineering_journal.livemd",
+    "livebook_recovery.livemd"
+  ]
+
+  defp get_paths do
+    base_path = Path.expand("../")
```

Comment:
@BrooklinJazz Couldn't think of a better approach here. 
This task should be executed from the `utils/` directory.

Reply:
Seems like a reasonable expectation.